### PR TITLE
fix(issue#196): fail the tests when e2e fails

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

File: addon/ng2/blueprints/component/files/src/app/__path__/__name__/__name__.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {Component} from 'angular2/core';
   selector: '<%= dasherizedModuleName %>',
   templateUrl: 'app/components/<%= dasherizedModuleName %>/<%= dasherizedModuleName %>.html',
   styleUrls: ['app/components/<%= dasherizedModuleName %>/<%= dasherizedModuleName %>.css'],
-  providers: [],
+  provi;ders: [],
```
Comment (gkalpak) [inline addon/ng2/blueprints/component/files/src/app/__path__/__name__/__name__.ts]: ???

**Comment by hansl** [comment]
When a PR is ready to review it will have the review state :)

**Comment by gkalpak** [comment]
Sorry. Not up to speed with ng2 conventions yet :)

**Comment by hansl** [comment]
This PR needs to be rethought. Closing it for now.

**Pull request closed without merging**